composer.json

```
@@ -7,48 +7,50 @@
     "prefer-stable": true,
     "require": {
         "php": ">=7.3",
-        "drupal/core": "^8.9.14",
-        "drupal/linkit": "~5.0-beta8",
-        "drupal/maxlength": "~1.0@beta",
+        "drupal/core": "^8.9.14 || ^9.1",
+        "drupal/linkit": "~5.0-beta12",
+        "drupal/maxlength": "~1.0-rc1",
         "easyrdf/easyrdf": "1.0.0 as 0.9.2",
         "openeuropa/oe_media": "~1.12",
-        "openeuropa/rdf_skos": "~1.0"
+        "openeuropa/rdf_skos": "1.x-dev"
     },
     "require-dev": {
         "composer/installers": "~1.5",
-        "drupal/core-composer-scaffold": "^8.8",
+        "drupal/core-composer-scaffold": "^8.9 || ^9",
         "drupal/address": "~1.7",
         "drupal/composite_reference": "~1.0-alpha2",
         "drupal/config_devel": "~1.2",
-        "drupal/datetime_testing": "1.x-dev",
-        "drupal/description_list_field": "1.0.x-dev",
+        "drupal/datetime_testing": "dev-3217303-drupal-9-compatibility",
+        "openeuropa/description_list_field": "1.0.x-dev",
```

```suggestion
        "drupal/description_list_field": "1.0.x-dev",
```
when module will be released.